### PR TITLE
TabPanel: get rid of use jsx-nobind ignore comment

### PR DESCRIPTION
**Pull request by EugeniyKiyashko**

### CODE REVIEW

File: JSDemos/Demos/TabPanel/SortableClosableTabs/React/App.js [hunk 1]
```diff
@@ -36,40 +35,41 @@ function App() {
     if (index >= newEmployees.length && index > 0) {
       setSelectedItem(newEmployees[index - 1]);
     }
-  }
+  }, [employees, setEmployees, setSelectedItem]);
 
-  function renderTitle(data) {
+  const renderTitle = React.useCallback((data) => {
     function closeHandler() {
```
Comment (ksercs) [inline JSDemos/Demos/TabPanel/SortableClosableTabs/React/App.js]: is we pass an arrow function to `onClick` why we need this function? Cannot we just pass `() => { closeButtonHandler(data); }`

File: JSDemos/Demos/TabPanel/SortableClosableTabs/React/App.js [hunk 2]
```diff
@@ -36,40 +35,41 @@ function App() {
...
           <span>
             {data.FirstName} {data.LastName}
           </span>
-          {employees.length >= 2 && <i className="dx-icon dx-icon-close" onClick={closeHandler} />}
+          {employees.length >= 2 && <i className="dx-icon dx-icon-close" onClick={() => { closeHandler(); }} />}
         </div>
       </React.Fragment>
     );
-  }
+  }, [closeButtonHandler]);
```
Comment (ksercs) [inline JSDemos/Demos/TabPanel/SortableClosableTabs/React/App.js]: shouldn't `employees` be in deps?

Comment (EugeniyKiyashko) [reply JSDemos/Demos/TabPanel/SortableClosableTabs/React/App.js]: You're right! Great job! https://github.com/DevExpress/devextreme-demos/pull/1916/commits/15f8186a1bee53602904eb8267b2fcd2f0b6b1f2

File: JSDemos/Demos/TabPanel/SortableClosableTabs/React/App.js (hunk 1 above)
Comment (EugeniyKiyashko) [reply JSDemos/Demos/TabPanel/SortableClosableTabs/React/App.js]: Yes, we can pass function directly. It is a rudiment from previous version. Thanks.
https://github.com/DevExpress/devextreme-demos/pull/1916/commits/15f8186a1bee53602904eb8267b2fcd2f0b6b1f2

**Review by Zedwag — DISMISSED** [review]

File: JSDemos/Demos/TabPanel/SortableClosableTabs/React/App.js [hunk 3]
```diff
@@ -36,40 +35,35 @@ function App() {
...
+        {employees.length >= 2 && <i className="dx-icon dx-icon-close" onClick={() => { closeButtonHandler(data); }} />}
+      </div>
+    </React.Fragment>
+  ), [closeButtonHandler]);
 
-  function onSelectionChanged(args) {
+  const onSelectionChanged = React.useCallback((args) => {
     setSelectedItem(args.addedItems[0]);
-  }
+  }, [employees, setSelectedItem]);
```
Comment (ksercs) [inline JSDemos/Demos/TabPanel/SortableClosableTabs/React/App.js]: seems like you added `employees` deps to another function :)

**Review by ksercs — APPROVED** [review]

**Pull request merged**